ToDoApp/ToDoApp/CreateTaskView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  CreateTaskView.swift
+//  ToDoApp
+//
+//  Created by Trong Hieu Vuong on 2022-12-01.
+//
+
+import Foundation
+import SwiftUI
+
+struct CreateTaskView: View {
+    @StateObject private var createTaskViewModel = CreateTaskViewModel()
+   var body: some View {
+       VStack(spacing: 15) {
+           NavigationStack{
+               VStack(alignment: .leading, spacing: 15){
+                   Text("Task Title")
+                       .multilineTextAlignment(.leading)
+                   TextField("Insert Title", text: $createTaskViewModel.titleInput)
+                       .padding(.leading, 10.0)
+                     .textInputAutocapitalization(.words)
+                     .frame(width: 390.0, height: 40)
+                     .overlay(
+                        RoundedRectangle(cornerRadius: 14)
+                            .stroke(Color.accentColor, lineWidth: 2)
+                     )
+                   Spacer()
+                       .frame(height: 15)
+                   Text("Date of Completion")
+                   Button("Choose Date"){
```

Comment:
You just need to use DatePicker here. it will look like a text field until it is tapped. once it is tapped the date selection UI will appear automatically
here is a link on how to use it
https://www.hackingwithswift.com/quick-start/swiftui/how-to-create-a-date-picker-and-read-values-from-it